src/mqttcm_connect.c

```diff
@@ -56,6 +56,12 @@ int get_global_shutdown()
 	return 0;
 }
 
+int Get_mqinit()
+{
+    MqttCMInfo("Inside Get_mqinit weak function.\n");
+    mqinit = 0;
```

Comment:
Adding mqinit = 0 is not correct in getter func as the value will get changed to 1 or 0 during process flow